messages/main.proto

```diff
@@ -793,3 +794,23 @@ message ShutdownScheduled
     ShutdownReason shutdown_reason = 1;
     optional uint32 ms_until_shutdown = 2;
 }
+
+message Polarizer
+{
+    enum Command {
+        // perform homing procedure which last a few seconds
+        // during that time, no command can be accepted
+        POLARIZER_HOME = 0;
+        // commands to set a specific glass
+        POLARIZER_PASS_THROUGH = 1;
+        POLARIZER_0_HORIZONTAL = 2;
+        POLARIZER_90_VERTICAL = 3;
+
+        // see angle below
+        POLARIZER_CUSTOM_ANGLE = 4;
+    }
+    Command command = 1;
+    uint32 angle = 2; // with `POLARIZER_CUSTOM_ANGLE` only, [0, 360º], 0º =
```

Comment:
we can't go more precise than 1º ? I am not sure that we need it tbh, but just wondered about resolutiion

Reply:
i don't think we care because `angle` should only be for debugging purposes

Reply:
done :)

Reply:
ty! Though I agreed with you that I don't think we would need more than 1 degree res. for debugging :D